### PR TITLE
Add login with email

### PR DESCRIPTION
**Pull request by dirodriguezm**
- Login with email
- Logout
- Refresh token funcionality

### CODE REVIEW

File: components/loginForm.vue [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+                :append-icon="showPassword ? 'mdi-eye' : 'mdi-eye-off'"
+                :type="showPassword ? 'text' : 'password'"
+                @click:append="showPassword = !showPassword"
+              ></v-text-field>
+            </v-col>
+            <v-col v-if="afterRegister" cols="12">
+              <v-alert icon="mdi-email" color="blue-grey" dismissible>
+                Registered user successfully. Check the email we sent to
+                <strong>{{ userData.email }}</strong> to activate the account.
+                <br />Enjoy ALeRCE Watchlist.
```
Comment (JavierArredondo) [inline components/loginForm.vue:35]: ALeRCE Explorer

Comment (JavierArredondo) [reply components/loginForm.vue:35]: or no?

**Review by JavierArredondo — COMMENTED** [review]
Look nice for me. I cloned the changes to test it

File: components/loginForm.vue [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+            </v-col>
+          </v-row>
+          <v-row>
+            <v-col cols="12">
+              <v-btn
+                id="googleLogin"
+                :loading="isLoading"
+                :disabled="isLoading"
+                color="primary"
+                class="ma-2 white--text"
```
Comment (JavierArredondo) [inline components/loginForm.vue:68]: Maybe remove `ma` from class property (visual improve)

File: components/loginForm.vue (hunk 1 above)
Comment (dirodriguezm) [reply components/loginForm.vue:35]: Upsi

**Review by JavierArredondo — APPROVED** [review]

**Pull request merged**